apps/vanilla/src/global/components.css

```
@@ -1,28 +1,314 @@
-a,
-:link,
-.link {
-  --link-color: var(--primary-color);
+.default-layout {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	font: var(--body-font);
+}
+
+.main-wrapper {
+	max-width: 1360px;
+	display: flex;
+	flex-direction: column;
+	margin: 0 24px;
+}
+
+/* Header styles */
 
-  color: var(--link-color);
-  text-decoration: none;
-  border-bottom: 1px solid var(--link-color);
+.books-app-header {
+	display: flex;
+	align-items: center;
+	font: var(--logo-font);
+	height: 64px;
+	border-bottom: 1px solid rgb(0 0 0 / 15%);
+	position: sticky;
+	top: 0;
+	background-color: var(--surface-color);
+
+	/* z-index is used because books-group__title with position: sticky create new overlay context */
+	z-index: 1;
 }
 
-a:hover,
-:link:hover,
-.link:hover {
-  --link-color: var(--primary-light-color);
+/* Main styles */
+
+.books-app-content {
+	display: flex;
+	gap: 24px;
 }
 
-.links {
-  display: flex;
-  flex-wrap: wrap;
-  gap: 16px;
+.books-group {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	gap: 24px;
 }
 
-.default-layout {
-  --default-layout-width: 600px;
+.books-group__title {
+	display: flex;
+	gap: 8px;
+	position: sticky;
+	top: 65px;
+	height: 24px;
+	width: 100%;
+	padding: 16px 0;
+	background-color: var(--surface-color);
+	font: var(--subtitle-font);
+	text-transform: var(--uppercase);
+}
+
+.books {
+	display: grid;
+	grid-template-columns: repeat(auto-fill, minmax(180px, 1fr));
+	max-width: 900px;
+	gap: 16px;
+}
+
+.books p {
+	max-height: 64px;
+	overflow: hidden;
+	-webkit-box-orient: vertical;
+	-webkit-line-clamp: 2;
+	/* stylelint-disable-next-line value-no-vendor-prefix */
+	display: -webkit-box;
+	font: var(--body-small-font);
+}
+
+.book__image {
+	width: 100%;
+	border-radius: 24px;
+}
+
+.filters-wrapper {
+	width: 320px;
+	position: relative;
+}
+
+.filters__header {
+	font: var(--subtitle-font);
+	text-transform: var(--uppercase);
+}
+
+.filters {
+	width: 320px;
+	display: flex;
+	flex-direction: column;
+	gap: 24px;
+	position: sticky;
+	top: 65px;
+	padding-top: 16px;
+}
+
+.filters__inner-wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: 24px;
+}
+
+.search-input {
+	height: 48px;
+	padding-left: 48px;
+	border-radius: 16px;
+	border: none;
+	background-color: var(--surface-overlay-color);
+	background-image: url("../pages/static/heroicons_magnifying-glass.png");
+	background-repeat: no-repeat;
+	background-position: 12px 12px;
+	font: var(--body-font);
+}
+
+::placeholder {
+	color: var(--on-surface-dim-color);
+}
+
+.select-input {
+	height: 48px;
+	width: 100%;
+	border-radius: 16px;
+	border: none;
+	background-color: var(--surface-overlay-color);
+	font: var(--body-font);
+	padding-left: 16px;
+	appearance: none;
+	background-image: url("../pages/static/heroicons_chevron-down.png");
+	background-repeat: no-repeat;
+	background-position: 95% 12px;
```

This way it'll always be `12px` from the right edge.  
```suggestion
  background-position: calc(100% - 12px) 12px;
```